docker/Dockerfile

```diff
@@ -28,9 +28,11 @@ RUN \
     # Ok, cleanup!
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
-    rm -rf /tmp/* && \
+    rm -rf /tmp/*
+
+Run \
     mkdir -p /srv/www /root /var/run/s6 /etc/services.d/snap && \
-    cp docker/run_snap /etc/services.d/snap/run && \
+    cp /srv/www/run_snap /etc/services.d/snap/run && \
     rm -rf /etc/services.d/node
```

Comment:
This cna be part of a single step, which makes the image a bit smaller. I'm also not convinced there is a /srv/www/run_snap to copy.